superset-frontend/src/datasource/DatasourceEditor.jsx

```diff
@@ -261,7 +262,11 @@ StackedField.propTypes = {
 };
 
 function FormContainer({ children }) {
-  return <Well style={{ marginTop: 20 }}>{children}</Well>;
+  return (
+    <Card style={{ marginTop: 20 }} bodyStyle={{ padding: 15 }}>
```

Comment:
Can you move the inline styles to Emotion?

Reply:
There 2 cases of **Card** component where one requires different styling than the other. Default **Card** styling is in the Emotion already

Reply:
It looks like we might not really _need_ that 20px top margin. 

As for the bodyStyle's padding, I would love to use the Theme variables here too. There are a few options:

1) use `withTheme` to use the `gridUnit`s here directly (4 gridUnits for 16px would be close enough)
2a) remove the bodyStyle prop and use a **class name** (e.g. 'padded') and use that className in the component's Emotion stylesheet to add the padding (again, with `gridUnit`s)
2b) remove the bodyStyle prop and use a **prop** (e.g. 'padded') and use that prop in the component's Emotion stylesheet to apply the padding (again, with `gridUnit`s)

I'd prefer either 2a or 2b, personally.